README.md

```diff
@@ -250,6 +250,27 @@ $ remme public-key get-info \
 }
 ```
 
+Get a state by its address — ``remme state get``:
+
+| Arguments | Type   | Required | Description                        |
+| :-------: | :----: | :------: | ---------------------------------- |
+| address   | String | Yes      | Account address to get a state by. |
+| node-url  | String | No       | Node URL to apply a command to.    |
+
+```bash
+$ remme state get \
+      --address=0000000000000000000000000000000000000000000000000000000000000000000001 \
```

Comment:
Why did you use this address? Is this an account address?

Reply:
@anastasiia-bilova, done.

Reply:
Use your real account address. In tests too.

Reply:
@anastasiia-bilova, what will it change?

Reply:
This address `0000000000000000000000000000000000000000000000000000000000000000000001 ` doesn't say anything. You must show which address you use, address `account` or` node_account`.

Reply:
I understand, but what a difference between returned data `account type address` and `node account`?
In README I have changed the address for more clarity.

Reply:
There is no difference, just for the perception that you are using a real address.